models/questions.py

```diff
@@ -15,10 +15,10 @@
 
 # In SQL we can manually add the constraint
 # alter table questions add constraint name_bc_unique UNIQUE(name, base_course);
-try:
-    db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
-except:
-    pass
+# try:
+#     db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
+# except:
+#     pass
```

Comment:
I was getting an error when there is another migration to be done later in the file. The attempt to run db.executesql seems to poison the session objec that will be needed later, so try/except wasn't sufficient. (Also seems inefficient to keep running this query every time web2py loads. Isn't there a way to get this constraint through web2py define_table?)